pkg/registry/data/registry.json

```diff
@@ -546,6 +596,130 @@
       ],
       "transport": "stdio"
     },
+    "brightdata-mcp": {
+      "description": "An MCP interface into the Bright Data toolset for web scraping and data extraction",
```

Comment:
nit: In the existing MCP servers the properties are in alphabetical order, first "args" then "description" then "env_vars" etc